inc/extras.php

```diff
@@ -15,15 +15,17 @@
  * @return array
  */
 function siteorigin_corp_body_classes( $classes ) {
```

Comment:
I just noticed we're not applying the hentry fix ([reference](https://github.com/siteorigin/siteorigin-north/blob/develop/functions.php#L159)). Can you add it to this PR real quick?

Reply:
https://github.com/siteorigin/siteorigin-corp/blob/release/0.8/inc/extras.php#L68-L89

Reply:
Awesome.

Real talk: I mistook body_class as post_class. ;(

Reply:
Better we check. Good work.